semantic_conventions/resource/cloud.yaml

```
@@ -99,6 +105,9 @@ groups:
             - id: azure_app_service
               value: 'azure_app_service'
               brief: Azure App Service
+            - id: azure_openshift
+              value: 'azure_openshift'
+              brief: Red Hat OpenShift on Azure Cloud
```

```suggestion
              brief: Azure Red Hat OpenShift
```

I'm not sure what we should put on `brief` but it looks like the "product name" to me. I saw for Alibaba you used the exact same name as state on their website, but for the others is a bit different. Should we stay with the "advertised" service/product name on the respective pages? If so, then the others also need to be adjusted.

---

Ive here no strong opinion. Let me change that :+1:

---

i compared it with the others. Now it should fit :+1: